strawberry_django_plus/types.py

```diff
@@ -212,6 +213,17 @@ def resolve_model_field_type(
             doc = field_type.__doc__ and inspect.cleandoc(field_type.__doc__)
             enum_def = strawberry.enum(field_type, description=doc)._enum_definition
         retval = enum_def.wrapped_cls
+
+    elif getattr(field, "choices", None) is not None:
+        enum_class_fields = {c[0]: c[0] for c in field.choices}
+        auto_enum = enum.Enum(
+            f"{field.model._meta.object_name}{capitalize_first(to_camel_case(field.name))}AutoEnum",
```

Comment:
One problem I see here is that, if we use only the object_name, two objects with the same name would produce the same enum name here. So probably the app name also needs to be taken in consideration?

Also, I think this should be configurable. We have a settings module here which can be used to define it the user wants or not to auto generate enums, since not everyone will like this.